config/locales/en.yml

```diff
@@ -188,6 +190,13 @@ en:
             yes: Yes
             no: No
             save: Save
+      agreements:
+        title: Legal Agreements
+        notice: "These are legal agreements that are required to be reviewed and agreed upon by all applicants of %{hackathon_name}."
+        name: Name
```

Comment:
Let's make the Agreement URL a single-line input textbox, instead of a multiline field, to match behavior in other parts of HM (i.e., config for assets that ask for a URL) -- so this should prob be updated in the migration!